examples/pytorch/swav_queue.py

```diff
@@ -17,20 +17,24 @@ def __init__(self, backbone):
         super().__init__()
         self.backbone = backbone
         self.projection_head = SwaVProjectionHead(512, 512, 128)
-        self.prototypes = SwaVPrototypes(128, 512, 10)
+        self.prototypes = SwaVPrototypes(128, 512, 5)
 
-        self.start_queue_at_epoch = 15
+        self.start_queue_at_epoch = 2
         self.queues = nn.ModuleList([MemoryBankModule(size=3840) for _ in range(2)])
 
-    def forward(self, high_resolution, low_resolution, epoch=None, step=None):
+    def forward(self, high_resolution, low_resolution, epoch, step):
         self.prototypes.normalize()
 
         high_resolution_features = [self._subforward(x) for x in high_resolution]
         low_resolution_features = [self._subforward(x) for x in low_resolution]
 
-        high_resolution_prototypes = [self.prototypes(x, step) for x in high_resolution_features]
-        low_resolution_prototypes = [self.prototypes(x, step) for x in low_resolution_features]
-        queue_prototypes = self._get_queue_prototypes(high_resolution_features, epoch)
+        high_resolution_prototypes = [
+            self.prototypes(x, step) for x in high_resolution_features
+        ]
```

Comment:
I just checked the swav paper and they freeze the prototypes by epochs and not iterations. I guess we can just pass the epoch here instead of the step. Then we also don't have to count the steps in an extra variable.

Reply:
That's true. I'll change that in both examples.